FHIR-Converter/client/src/core/common/constants/template-management.ts

```diff
@@ -2,14 +2,25 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License in the project root for license information.
  */
+import { TemplateType } from '../enum/template-type'
 
+const OfficialLoginServer = 'healthplatformregistry.azurecr.io';
 const Hl7v2TemplatesImage = 'hl7v2defaulttemplates';
 const CcdaTemplatesImage = 'ccdadefaulttemplates';
-const OfficialLoginServer = 'healthplatformregistry.azurecr.io';
-export const Hl7v2SampleDataReference = `${OfficialLoginServer}/hl7v2sampledata:latest`;
-export const CcdaSampleDataReference = `${OfficialLoginServer}/ccdasampledata:latest`;
-export const Hl7v2ImageBaseReference = `${OfficialLoginServer}/${Hl7v2TemplatesImage}`;
-export const CcdaImageBaseReference = `${OfficialLoginServer}/${CcdaTemplatesImage}`;
+
 export const OfficialRepoTokenUrl = `https://${OfficialLoginServer}/oauth2/token?scope=repository:*:*&service=${OfficialLoginServer}`;
-export const Hl7v2TagsUrl = `https://${OfficialLoginServer}/acr/v1/${Hl7v2TemplatesImage}/_tags`;
-export const CcdaTagsUrl = `https://${OfficialLoginServer}/acr/v1/${CcdaTemplatesImage}/_tags`;
+
+export const ImageTagsUrls = {
+	[TemplateType.hl7v2]: `https://${OfficialLoginServer}/acr/v1/${Hl7v2TemplatesImage}/_tags`,
+	[TemplateType.ccda]: `https://${OfficialLoginServer}/acr/v1/${CcdaTemplatesImage}/_tags`
+};
+
+export const SampleDataImageReferences = {
+	[TemplateType.hl7v2]: `${OfficialLoginServer}/hl7v2sampledata:latest`,
+	[TemplateType.ccda]: `${OfficialLoginServer}/ccdasampledata:latest`
+};
+
+export const TemplateImageBaseReferences = {
+	[TemplateType.hl7v2]: `${OfficialLoginServer}/${Hl7v2TemplatesImage}`,
+	[TemplateType.ccda]: `${OfficialLoginServer}/${CcdaTemplatesImage}`
```

Comment:
Do we need to add urls and references for JSON as well?

Reply:
Though there's no default templates for JSON, we still provide customers with sample data and sample templates. They are treated the same way as the default templates, so the extension should be able to pull and push the "default" templates for JSON as well.